.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file is generated by eclispe. Normaly this file should exist ONLY on your local repo (on you computer). Think about it, maybe I don't use eclipse (I use IntelliJ IDE) so this file is not pertinent for me (or other developer). For this reason there is a '.gitignore' file  in you git repo. In this file we specify wich files we don't add to push on remote repo (files that we want that git IGNORES). You can specify also a file pattern in git ignore. Read a little bit about gitignore and his purpose. And add this file to gitignore so we don't track it on remote repo. On the web there exists tools to generate some gitignore file (https://www.gitignore.io/) , feel free to explore. You can specify here: java, eclipse, intellij and the tool we'll generate for this file. Still, at this stage is better to write your own .gitingnore file so you better understand. To complete this comment add this file(.classpath) to .gitignore.